brendenwest/lostappleproject
# Mg 31 applelist firebase display

This addresses Issue #31 .

Having successfully connected to the Firebase Firestore DB, I have added code that pulls the name from each apple and displays it in a FlatList, next to an Apple Icon image. This List of Apple Names is scrollable.

No new libraries are added--Styles created, and code added to the AppleList component page. Modified Header Styles slightly, due to flex sizing causing an issue.

To Verify:

1. `git fetch`, `git checkout mg-31-applist-firebase-display`.
2. Launch Metro `(npx react-native start`), and launch app in the emulator (`npx react-native run-android`).
3. Navigate to "Apples" page.

You should see the following:
![image](https://user-images.githubusercontent.com/49133101/139753294-7f65376c-cf19-4b7f-9315-127fdaeafad0.png)
